Review.java

```diff
@@ -164,5 +164,22 @@ public static String randomAdjective()
     } else {
       return randomNegativeAdj();
     }
+   public static int starRating(String fileName)
+   {
+      double totalValue = totalSentimentVal(fileName);
+      if(totalValue >= 1)
+      {
+         return 5;
+      }
+      else if(totalValue >= 0)
+      {
+         return 3;
+      }
+      else
+      {
+         return 1;
+      }
+    }

```

Comment:
The code looks well! Only thing I would add is a comment describing why you chose those values